sb_vision/tokens.py

```diff
@@ -183,10 +198,12 @@ def from_apriltag_detection(
         arr = [apriltag_detection.H.data[x] for x in range(9)]
         homography = np.reshape(arr, (3, 3))
 
+        id = apriltag_detection.id
```

Comment:
We should avoid shadowing a builtin if possible: `id` -> `id_`.

Reply:
I think i'd prefer `marker_id`, as it gives a better idea of what's its an id of